Tasks/Gradle/CodeAnalysis/Common/BaseTool.ts

```diff
@@ -46,26 +46,12 @@ export abstract class BaseTool implements IAnalysisTool {
      */
     protected abstract parseXmlReport(xmlReport: string, moduleName: string): [number, number];
 
-    public configureBuild(toolRunner: ToolRunner): ToolRunner {
-        switch (this.buildOutput.buildEngine) {
-            case BuildEngine.Maven: {
-                tl.debug("Not implemented");
-                throw new Error();
-            }
-            case BuildEngine.Gradle: {
-
-                if (this.isEnabled()) {
-                    console.log(tl.loc('codeAnalysis_ToolIsEnabled'), this.toolName);
-
-                    var initScriptPath: string = path.join(__dirname, '../', this.toolName.toLowerCase() + '.gradle');
-                    toolRunner.arg(['-I', initScriptPath]);
-                    break;
-                }
-            }
-        }
-        return toolRunner;
-    }
-
+    /**
```

Comment:
While there is no real cost to doing so, there is also no functional change to the Gradle task in this PR.